toolchain/sem_ir/formatter.cpp

```diff
@@ -70,28 +70,28 @@ class FormatterImpl {
     CloseBrace();
     out_ << '\n';
 
-    for (int i : llvm::seq(sem_ir_->interfaces().size())) {
-      FormatInterface(InterfaceId(i));
+    for (auto [id, _] : sem_ir_->interfaces().enumerate()) {
```

Comment:
These make me wonder if we want a `.keys()` or `.ids()` or similar accessor to get the IDs but not the values. But we can leave that for another day :)